yawning_titan/config/game_config/miscellaneous_config.py

```diff
@@ -26,6 +28,20 @@ def output_timestep_data_to_json(self) -> bool:
     def output_timestep_data_to_json(self, value):
         self._output_timestep_data_to_json = value
 
+    @property
+    def random_seed(self) -> bool:
```

Comment:
This should be an `int`, right? And `Optional` as the config allows for null value which is deserialized as `None`.

```suggestion
    def random_seed(self) -> Optional[int]:
```